### PR TITLE
Allow buttons to be branch links

### PR DESCRIPTION
**Pull request by palmfjord**
🛤 🌲

### CODE REVIEW

File: src/blocks/ImageTextBlock.tsx [hunk 1]
```diff
@@ -124,16 +127,31 @@ export const ImageTextBlock: React.FunctionComponent<ImageTextBlockProps> = ({
...
-            <ButtonLinkWithMargin
-              href={getStoryblokLinkUrl(button_link)}
-              type={button_type}
-              size="sm"
-              bold
-            >
-              {button_title}
-            </ButtonLinkWithMargin>
-          )}
+          {show_button &&
```
Comment (hedvigoscar) [inline src/blocks/ImageTextBlock.tsx]: Could it be desirable to split these down to `show_regular_button` and `show_branch_button` or something similar? This condition feels a bit weird as it is now.

Comment (hedvigoscar) [reply src/blocks/ImageTextBlock.tsx]: Or perhaps even an enum, `NO_BUTTON, REGULAR_BUTTON, BRANCH_BUTTON` 🙂

Comment (palmfjord) [reply src/blocks/ImageTextBlock.tsx]: I  see what you mean, unsure if i agree tho. It kinda makes sense to me to do the same in all blocks.

**Comment by palmfjord** [comment]
@hedvigoscar you got it 🙌

**Pull request merged**